roles/openshift_certificate_expiry/test/conftest.py

```diff
@@ -23,7 +23,10 @@
     {
         'short_name': 'combined',
         'cn': 'combined.example.com',
-        'serial': 6,
+        # Verify that HUGE serials parse correctly.
+        # Frobs PARSING_HEX_SERIAL in _parse_cert
+        # See https://bugzilla.redhat.com/show_bug.cgi?id=1464240
+        'serial': 14449739080294792594019643629255165375,
```

Comment:
Is this sufficient? The tested function notes `` # Hex serials arrive colon-delimited``